.github/workflows/main.yaml

```diff
@@ -9,9 +9,12 @@ on:
 
 jobs:
   deploy-staging:
-    uses: viam-modules/common-workflows/.github/workflows/deploy_training_script.yaml@main
+    uses: viam-modules/common-workflows/.github/workflows/deploy_training_script.yaml@modify-workflows
```

Comment:
We should leave this on main when we merge (I understand this was for testing purposes. Just calling out that we need to change back before merging)